bindings/python/tests/test_storage.py

```diff
@@ -57,6 +57,40 @@
 # Test query
 
 
+# Test from_bytes()
+mturk_bytes = '''{"9b256962-8c81-45f0-949c-c9d10b44050b": {
+  "meta": "http://onto-ns.com/meta/0.1/Person",
+  "dimensions": {"N": 2},
+  "properties": {
+    "name": "Mechanical Turk",
+    "age": 83,
+    "skills": ["chess", "cheating"]
+  }
+}}'''.encode()
+mturk = dlite.Instance.from_bytes("json", mturk_bytes)
```

Comment:
can you add a from_bytes with options in the test too?

Reply:
It is possible to add options, but they should not affect anything, since all the options in the json plugin are for controlling the output (from save() and to_bytes().

Reply:
Added a new test with options